openmdao/core/problem.py

```diff
@@ -1268,8 +1268,11 @@ def check_partials(self, out_stream=_DEFAULT_OUT_STREAM, includes=None, excludes
         excludes = [excludes] if isinstance(excludes, str) else excludes
 
         comps = []
+        under_CI = env_truthy('GITHUB_ACTION') or env_truthy('TRAVIS')
```

Comment:
I think we can just use the environment variable `CI` here..  it's boolean and available on GH Actions, Travis-CI and CircleCI:
https://docs.github.com/en/actions/learn-github-actions/environment-variables
https://docs.travis-ci.com/user/environment-variables
https://circleci.com/docs/env-vars

Reply:
fixed